### DISCUSSION TITLE
Gogs repositories

### DISCUSSION THREAD

**Discussion by BenBorre** [gogs/gogs#7603] [Q&A]
Why does Gogs seems to create only bare repositories? It is creating only repositories with a name like repname.git rather than creating a .git directory under a directory. This prevents me from being able to connect tools like git bash or gitkraken to it. Is there a way to proceed in another way? Many, may thanks, because I like this product very much.

**Comment by annpocoyo** [comment]
I know I am late but you aren't meant to edit the repositories directly but rather to `git pull` and `git push`.

However if you need to manually edit the repositories I suggest you clone a local copy and use that to push and pull like so `git clone path/to/repo.git`.
